rootfs/start.sh

```diff
@@ -59,7 +59,7 @@ listen main
 EOF
 
     # Add erb code to a new template file
-    sed "/^    bind \+\*\?:/s/\*\?:\(.*\)/<%= bind_tcp('0.0.0.0', \1) %>/" \
+    sed "/^    bind \+\*\?:/s/\*\?:\(\([0-9]*\)\|\({{[^}]*}}\)\)/<%= bind_tcp('0.0.0.0', \1) %>/" \
```

Comment:
This refactor was needed ensure that only the port was included in the `\1` regex capture group . Notice the two cases:
- `[0-9]*` will match actual ports, for example `443` in `bind *:443`
- `{{[^}]}}` will match when the port is templatized, for example `{{ $cfg.StatsPort }}` in `bind *:{{ $cfg.StatsPort }}`